x-pack/filebeat/modules.d/aws.yml.disabled

```diff
@@ -6,7 +6,7 @@
     enabled: true
 
     # AWS SQS queue url
-    #var.queue_url: sqs_queue_url
+    #var.queue_url: <sqs_queue_url>
```

Comment:
Why doing this change? We don't use to use this kind of placeholders in the reference config, but you could change it to contain an example as `var.queue_url: 'https://sqs.myregion.amazonaws.com/123456/myqueue'`

Reply:
Yeah I wanted to make sure users know sqs_queue_url is not some kind of variable and it should be replaced. But the example you gave is better! I will change to what you suggested! Thanks!